Assets/LeapMotion+OVR/Scripts/TouchCubeQueues.cs

```diff
@@ -25,6 +25,10 @@ public class TouchCubeQueues : MonoBehaviour {
   public GameObject warpImagesCanvas;
 
   private int demoStage = 0;
+
+  void Start () {
+    Reset ();
```

Comment:
Watch out, the Reset() name is one of the editor-time methods.  Specifically it is called if a user right clicks a script and selected Reset.  In this case it causes editor time errors.